### DISCUSSION TITLE
Econo mode not showing on Faikin UI

### DISCUSSION THREAD

**Discussion by electrocamuk** [revk/xESP32-Faikout#320] [Q&A]
Hi, I tested and added my Daikin "Siesta"  ATX35JV1B to the supported list, having great success with it so far, awesome work!
My unit supports econo mode (limits compressor max Hz) via the IR remote,  but I cannot see this option or how to enable it in the Faikin UI, the econo mode is also not present in the array sent over MQTT:-

```
{
  "protocol": "S21",
  "online": true,
  "home": 24.5,
  "heat": true,
  "fanrpm": 290,
  "comp": 0,
  "outside": 14,
  "liquid": 30,
  "anglev": 28,
  "power": false,
  "mode": "H",
  "temp": 23,
  "fan": "3",
  "swingv": false,
  "powerful": false,
  "autor": 0,
  "autot": 0,
  "auto0": "00:00",
  "auto1": "00:00",
  "autop": false
}

```
Thanks in advance for the help!
-Dave

**Comment by revk** [comment]
I need debug of the S21 messages really, we show it if the messages appear to work. So maybe the unit is being different somehow.

**Reply by electrocamuk** [reply]
What's the best windows program for logging MQTT messages in your view? thanks :)

**Comment by electrocamuk** [comment]
Ahh, so maybe if I place the unit in econo mode via the IR remote control, it might show up in the Faikin UI?  I'll look into getting debug messages, I saw an option for that in the advanced settings area.

**Comment by electrocamuk** [comment]
![Screenshot_20240505_114958_Chrome](https://github.com/revk/ESP32-Faikin/assets/128866103/9ac0d38a-1c4d-49e5-9831-ff33e77c5b31)
Something i have noticed that might be a fault with my unit is, yesterday was a fairly sunny day, no heating or cooling required, so the unit sat at room temp, but you can see the room temp differs wildly from the liquid line temp, and the ESP based DS18B20 temperature sensor thats in the room, indoor thermistor fault would you say? Thanks
